app/admin/models/Orders_model.php

```diff
@@ -154,9 +158,25 @@ public function scopeListFrontEnd($query, $options = [])
             }
         }
 
+        $search = trim($search);
+        if (strlen($search)) {
+            $query->search($search, $searchableFields);
+        }
+
+        if (($startDateTime = array_get($dateTimeFilter, 'orderDateTime.start', false)) AND ($endDateTime = array_get($dateTimeFilter, 'orderDateTime.end', false))) {
+            $query = $this->scopeWhereBetweenOrderDateTime($query, date('Y-m-d H:i:s', $startDateTime), date('Y-m-d H:i:s', $endDateTime));
+        }
```

Comment:
Ooops, forgot to mention, use Carbon instead `Carbon::parse(startDateTime)->format()` or `make_carbon(startDateTime)->format()`

Also should be `start_at` and snake_case like you intended ;)